dhall-docs/src/Dhall/Docs/CodeRenderer.hs

```diff
@@ -102,18 +102,18 @@ data JtdInfo
         * @jtdInfoA@ and @jtdInfoB@ are the associated 'JtdInfo' inferred from
         @foo@ and @bar@
     -}
-    = RecordFields (Set.Set VarDecl)
+    = RecordFields (Set.Set NameDecl)
     -- | Default type for cases we don't handle
     | NoInfo
     deriving (Eq, Ord, Show)
 
--- | To make each variable unique we record the source position where it was
+-- | To make each name unique we record the source position where it was
 --   found.
-data VarDecl = VarDecl Src Text JtdInfo
+data NameDecl = NameDecl Src Text JtdInfo
```

Comment:
It might be useful to give a few examples of what these `NameDecl`s can be: let-bindings, record entries etc.